src/service/video/docking.js

```diff
@@ -1417,41 +1546,86 @@ export class VideoDocking {
         this.getTopEdge_() + margin :
         this.getBottomEdge_() - margin - targetHeight);
 
-    return {x, y, targetWidth, targetHeight};
+    const initialY = this.calculateInitialY_(
+        posY, this.getTopEdge_(), this.getBottomEdge_(), height);
+
+    return {x, y, targetWidth, initialY};
   }
 
   /**
    * @param {!../../video-interface.VideoOrBaseElementDef} video
-   * @param {!RelativeX} posX
-   * @param {!RelativeY} posY
+   * @param {!AmpElement} slot
+   * @return {{x: number, y: number, targetWidth: number, initialY: number}}
```

Comment:
nit: consider adding a `typedef` for this structure to help the code self-document

Reply:
Done.